core/src/test/scala/com/salesforce/op/stages/impl/regression/RegressionModelSelectorTest.scala

```diff
@@ -120,9 +121,32 @@ class RegressionModelSelectorTest extends FlatSpec with TestSparkContext
 
   it should "set the data splitting params correctly" in {
     val modelSelector = RegressionModelSelector()
-    modelSelector.splitter.get.setReserveTestFraction(0.1).setSeed(11L)
+    modelSelector.splitter.get.setReserveTestFraction(0.1).setSeed(11L).setMaxTrainingSample(1000)
+
     modelSelector.splitter.get.getSeed shouldBe 11L
     modelSelector.splitter.get.getReserveTestFraction shouldBe 0.1
+    modelSelector.splitter.get.getMaxTrainingSample shouldBe 1000
+  }
+
+  it should "set maxTrainingSample and down-sample" in {
```

Comment:
Did you also mean to check `maxTrainingSample` was set in this test as well?

Reply:
I moved testing the maxTrainingSample set/get functions to [here](https://github.com/salesforce/TransmogrifAI/pull/413/files/8e2778da356242d0cd10d259282fb72709cbba8c#diff-a915d7f576999b5aa0cfe9b9d3e73d9bR128) and forgot to rename this test. Great catch!